Scheme.md

```diff
@@ -0,0 +1,89 @@
+# `isolated-app:` Scheme Explainer
+
+This document provides a more detailed look at the `isolated-app:` scheme, which is part of the proposal for [Isolated Web Apps](./README.md).
+Isolated Web Apps are a new proposal to build an application using web technologies, but with additional security properties compared to normal web pages.
+Isolated Web Apps bundle all their contents inside a web bundle that is then [signed to make its integrity verifiable](https://github.com/WICG/webpackage/blob/main/explainers/integrity-signature.md).
+A key difference between Isolated Web Apps and normal web pages is that Isolated Web Apps do not rely on DNS name resolution and HTTPS certificate authorities.
+Instead, they need to be downloaded and installed.
+A user agent can verify the integrity of an Isolated Web App by checking the signature and comparing its corresponding public key to a list of known trusted public keys.
+
+## `isolated-app:` Scheme
+
+Since Isolated Web Apps do not use HTTPS certificate authorities and also need to be independent of domains and name resolution, a new URL scheme is needed to signify this difference.
+The tentative name of this new scheme is `isolated-app:`, highlighting how Isolated Web Apps are isolated from each other and the web through enforced Content Security Policy and Cross-Origin Isolation.
+When loading an `isolated-app:` URL, the user agent checks the signatures of the underlying web bundle and whether it trusts the public key that was used to sign it.
+A user agent might trust, for example, public keys configured via enterprise policy, public keys of known distribution mechanisms/stores, or separately allow-listed public keys of individual Isolated Web Apps.
+`isolated-app:` URLs are considered Secure Contexts, just like HTTPS pages, and thus have access to APIs like Service Workers. `isolated-app:` URLs look like so:
+
+```
+isolated-app://signed-web-bundle-id/path/inside/app.js?some-query#foo
+^           ://^                   /^                 ?^         #^
+scheme         opaque host          path               query      fragment
+```
+
+### Host
+
+URLs with the `isolated-app:` scheme use a Signed Web Bundle ID (see next section) as their opaque host, as defined in the [URL standard](https://url.spec.whatwg.org/).
+The Signed Web Bundle ID being the host adds the requirement that it must be a valid hostname (see [RFC 3986](https://www.rfc-editor.org/rfc/rfc3986)), which is the case given that Signed Web Bundle IDs are base32 encoded without padding.
```

Comment:
```suggestion
The Signed Web Bundle ID being the host adds the requirement that it must be a valid hostname (see [RFC 3986](https://www.rfc-editor.org/rfc/rfc3986)), which is the case given that Signed Web Bundle IDs are [base32-encoded](https://tools.ietf.org/doc/tcllib/html/base32.html) without padding.
```

Reply:
I added a link to https://datatracker.ietf.org/doc/html/rfc4648 instead, which I think is a better reference.

Reply:
Perfect.